examples/fizzbuzz.rye

```diff
@@ -0,0 +1,59 @@
+; Fizz Buzz by @otobrglez - <otobrglez@gmail.com>
+print "--- --- --- Fizz Buzz --- --- ---"
+
+; function mod(number, divider){
```

Comment:
As there is no [Modulo operation](https://en.wikipedia.org/wiki/Modulo_operation) in Rye yet (to my knowledge) I tried to implement this one.